guides/source/active_support_core_extensions.md

```diff
@@ -2797,7 +2797,7 @@ XML_TYPE_NAMES = {
 
 By default the root node is "hash", but that's configurable via the `:root` option.
 
-The default XML builder is a fresh instance of `Builder::XmlMarkup`. You can configure your own builder with the `:builder` option. The method also accepts options like `:dasherize` and friends, they are forwarded to the builder.
+The default XML Builder is a fresh instance of `Builder::XmlMarkup`. You can configure your own builder with the `:builder` option. The method also accepts options like `:dasherize` and friends, they are forwarded to the builder.
```

Comment:
I think this refers to _a_ XML builder, not the XML Builder template format or the class to delegate to. We also don't upcase `builder` in the following lines. So I think we should keep this as is.